packages/browserslist-config-odyssey/index.ts

```diff
@@ -10,34 +10,27 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-function literalTuple<
-  T extends string,
-  U extends [ T, ...T[] ]
-> (
-  ...args: U
-): U {
+function literalTuple<T extends string, U extends [T, ...T[]]>(...args: U): U {
```

Comment:
hmmmm don't love this formatting but meh that's prettier I guess? Lets add an entry to our Office hours discussion list to review the prettier format in detail for typescript